xml_converter/generators/proto_templates/node.proto

```diff
@@ -0,0 +1,274 @@
+syntax = "proto3";
+import "google/protobuf/any.proto";
+
+package Proto_Node;
```

Comment:
We are not generating java code so we can omit the `package` element here. Better to leave it not defined in case we ever need to define it in the future.

Reply:
Removed